### PR TITLE
Updated release notes for v142

### PR DESCRIPTION
**Pull request by jyaistMap**
@DavidJVitale @AtmaMani Here are the release notes for changes since the v141 tag.

### CODE REVIEW

File: guide/02-api-overview/release-notes.ipynb [hunk 1]
```diff
@@ -1,5 +1,33 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# What's new in version 1.4.2\n",
+    "#### New\n",
+    "* Added ability to work with [StoryMap Journals](https://storymaps.arcgis.com/en/app-list/map-journal/) \n",
```
Comment (AtmaMani) [inline guide/02-api-overview/release-notes.ipynb]: Rephrase to "Added a new submodule [`storymap`](https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.apps.storymap.html) under the `arcgis.apps` module to work with [ArcGIS Story Maps](https://storymaps.arcgis.com/en/app-list/map-journal/)

File: guide/02-api-overview/release-notes.ipynb [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+    "* Added support for [registering data](http://enterprise.arcgis.com/en/server/latest/publish-services/windows/overview-register-data-with-arcgis-server.htm) and adding [`big data files shares`](http://enterprise.arcgis.com/en/server/latest/get-started/windows/what-is-a-big-data-file-share.htm) from Cloud Stores\n",
+    "* Added `bulk_update()` function to `ContentManager` to update properties on a collection of items\n",
+    "* New [Orthomapping](http://pro.arcgis.com/en/pro-app/help/data/imagery/introduction-to-ortho-mapping.htm) module\n",
```
Comment (AtmaMani) [inline guide/02-api-overview/release-notes.ipynb]: "Added a" new [orthomapping](...) module

File: guide/02-api-overview/release-notes.ipynb [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+    "* Enhanced [`clone()`](https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.gis.toc.html#arcgis.gis.ContentManager.clone_items) of [`feature layer views`](https://doc.arcgis.com/en/arcgis-online/manage-data/create-hosted-views.htm) to support varied rendering and template definitions\n",
+    "* Improved `clone()` of Pro Maps and Survey123 Forms\n",
+    "* Added support fro `clone()` of [Vector Tile Style Items](https://www.esri.com/arcgis-blog/products/developers/mapping/design-custom-basemaps-with-the-new-arcgis-vector-tile-style-editor/)\n",
```
Comment (AtmaMani) [inline guide/02-api-overview/release-notes.ipynb]: typo on "for"

File: guide/02-api-overview/release-notes.ipynb [hunk 4]
```diff
@@ -1,5 +1,33 @@
...
+    "* Improved performance when using `clone()` on `feature services`\n",
+    "* Added `default_aggregation_styles` environment settign to [`gis.env`](https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.env.html)\n",
```
Comment (AtmaMani) [inline guide/02-api-overview/release-notes.ipynb]: typo in "setting"

File: guide/02-api-overview/release-notes.ipynb [hunk 5]
```diff
@@ -1,5 +1,33 @@
...
+    "* Improved handling of data fields with the [`Spatial DataFrame`](https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.features.toc.html#spatialdataframe)\n",
```
Comment (AtmaMani) [inline guide/02-api-overview/release-notes.ipynb]: I think its "date" fields, not data. Could you double check?

**Comment by jyaistMap** [comment]
Thanks for the second set of eyes, @AtmaMani. Made the changes.

**Comment by jyaistMap** [comment]
Think I got them all now, @AtmaMani

**Pull request merged**